src/decompiler-naming.h

```diff
@@ -173,21 +175,9 @@ void RenameAll(Module& module) {
   // identifiers.
   // FIXME: this can obviously give bad results if the input is not C++..
   std::set<string_view> filter = {
-    { "const" },
-    { "std" },
-    { "allocator" },
-    { "char" },
-    { "basic" },
-    { "traits" },
-    { "wchar" },
-    { "t" },
-    { "void" },
-    { "int" },
-    { "unsigned" },
-    { "2" },
-    { "cxxabiv1" },
-    { "short" },
-    { "4096ul" },
+      {"const"},    {"std"},   {"allocator"}, {"char"},  {"basic"},
+      {"traits"},   {"wchar"}, {"t"},         {"void"},  {"int"},
+      {"unsigned"}, {"2"},     {"cxxabiv1"},  {"short"}, {"4096ul"},
```

Comment:
Again, I think this single-line-per-entry has some value maybe?

Reply:
Wrapped in `// clang-format off` and `on`.